src/test/scala/com/gu/support/promotions/SerialisationSpec.scala

```diff
@@ -0,0 +1,86 @@
l Pack subscription, and maintenance of that subscription for at least 35 consecutive days, customers will be eligible to receive a £10 M&S e-gift card.\n10. You will be contacted by Guardian News & Media within 35 days of your eligible subscription commencing to be given details of how to select and claim your Free Gift. You will need to claim your Free Gift within 90 days of receiving your redemption information via email.\n11. Please note that any terms and conditions which apply to the Free Gift are separate to the contract you will enter into when you purchase a subscription as referred to in paragraphs 4 and 6 above.\n12. The Free Gift cannot be exchanged or transferred by you and cannot be redeemed by you for cash or any other free gift or prize. You must pay all other costs associated with the Free Gift and not specifically included in the Free Gift as specified in these terms and conditions.\n13. We retain the right to substitute the Free Gift with another free gift of similar value in the event that the Free Gift is not available for any reason.\n14. Only one entry to this Promotion per person. Entries on behalf of another person will not be accepted.\n15. We take no responsibility for entries that are lost, delayed, misdirected or incomplete or cannot be delivered or entered for any technical or other reason. Proof of delivery of the entry is not proof of receipt. \n16. The Promoter of the Promotion is Guardian News & Media Limited whose address is Kings Place, 90 York Way, London N1 9GU. Any complaints regarding the Promotion should be sent to this address.\n17. Nothing in these terms and conditions shall exclude the liability of GNM for death, personal injury, fraud or fraudulent misrepresentation as a result of its negligence.\n18. GNM accepts no responsibility for any damage, loss, liabilities, injury or disappointment incurred or suffered by you as a result of entering the Promotion or accepting the Free Gift or any substitute if applicable. GNM further disclaims liability for any injury or damage to you or any other person’s computer relating to or resulting from participation in the Promotion. \n19. GNM reserves the right at any time and from time to time to modify or discontinue, temporarily or permanently, this Promotion with or without prior notice due to reasons outside its control (including, without limitation, in the case of anticipated, suspected or actual fraud). The decision of 20. GNM in all matters under its control is final and binding.\nGNM shall not be liable for any failure to comply with its obligations where the failure is caused by something outside its reasonable control. Such circumstances shall include, but not be limited to, weather conditions, fire, flood, hurricane, strike, industrial dispute, war, hostilities, political unrest, riots, civil commotion, inevitable accidents, supervening legislation or any other circumstances amounting to force majeure.\n21. The Promotion will be governed by English law.")
```

Comment:
Minor: I would probably put this in a separate object (and val) and refer to it in this test.